downloads/swan-lake-release-notes/swan-lake-2201.7.1/RELEASE_NOTE.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.7.1 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-7-1/
+active: 2201-7-1
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-7-1
+    - /downloads/swan-lake-release-notes/2201.7.1/
+    - /downloads/swan-lake-release-notes/2201-7-1-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-7-1-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.7.1
+
+<em>Swan Lake 2201.7.1 is the first patch release of Ballerina 2201.7.0 (Swan Lake Update 7) and it includes a new set of bug fixes to the language, standard library, and tooling.</em>
```

Comment:
```suggestion
<em>Swan Lake 2201.7.1 is the first patch release of Ballerina 2201.7.0 (Swan Lake Update 7) and it includes a new set of bug fixes to the language and standard library.</em>
```
Seems like we do not have any updates on tooling. Add this back if we do.

Reply:
Update RN with tooling section